ocamlformat-help.txt

```diff
@@ -168,7 +157,7 @@ OPTIONS
        --intf
            Parse file with unrecognized extension as an interface.
 
-       -n N, --max-iters=N (absent=10 or OCAMLFORMAT_MAX_ITERS env)
+       -n N, --max-iters=N
```

Comment:
Is there a good way to keep the doc for the default?

Reply:
I slightly modified the generated doc for int options and choice options (I think it does not make much sense for flags because they are not set by default).